sdk/core/System.ClientModel/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.1.0-beta.5 (Unreleased)
-
-### Features Added
+## 1.1.0-beta.5 (2024-07-11)
 
 ### Breaking Changes
```

Comment:
We generally do not classify changes between betas as "Breaking changes" in the change log.    The rationale behind that is that we don't offer guarantees about beta stability, and we don't want to have a lot of breaking changes listed out that may it harder to find breaking changes for GA builds.

Can we move these to `Other Changes`?

Reply:
Ah, I did not know that!  I can make it an added feature instead.

Reply:
Updated other instances of "Breaking changes" headers in the changelog to "Other changes" as well.  I reworked this one to indicate additions under "Features added" and put removals in "Other"